<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wifi2Status</class>
 <widget class="QDialog" name="Wifi2Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WIFI Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="wifi_client_status_group">
     <property name="title">
      <string>Client</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="wifi_client_status_grid">
        <item row="0" column="0">
         <widget class="QLabel" name="wifi_client_status_enabled_label">
          <property name="text">
           <string>Enabled:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="wifi_client_status_ip">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="wifi_client_status_enabled">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="wifi_client_status_status">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="wifi_client_status_subnet_mask">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="wifi_client_status_signal_strength">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="wifi_client_status_signal_strength_label">
          <property name="text">
           <string>Signal Strength:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="wifi_client_status_ip_label">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="wifi_client_status_subnet_mask_label">
          <property name="text">
           <string>Subnet Mask:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="wifi_client_status_gateway_label">
          <property name="text">
           <string>Gateway:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="wifi_client_status_status_label">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="wifi_client_status_mac_address_label">
          <property name="text">
           <string>MAC Address:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="wifi_client_status_rx_label">
          <property name="text">
           <string>RX Count:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="wifi_client_status_tx_label">
          <property name="text">
           <string>TX Count:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="wifi_client_status_stretch">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="wifi_client_status_gateway">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="wifi_client_status_mac_address">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="wifi_client_status_rx">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="wifi_client_status_tx">
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="wifi_ap_status_group">
     <property name="title">
      <string>Access Point</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="wifi_ap_status_grid">
        <item row="7" column="0">
         <widget class="QLabel" name="wifi_ap_status_tx_label">
          <property name="text">
           <string>TX Count:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="wifi_ap_status_enabled_label">
          <property name="text">
           <string>Enabled:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="wifi_ap_status_mac_address_label">
          <property name="text">
           <string>MAC Address:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="wifi_ap_status_gateway_label">
          <property name="text">
           <string>Gateway:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="wifi_ap_status_connected_count_label">
          <property name="text">
           <string>Connected Count:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="wifi_ap_status_ip_label">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="wifi_ap_status_subnet_mask_label">
          <property name="text">
           <string>Subnet Mask:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="wifi_ap_status_rx_label">
          <property name="text">
           <string>RX Count:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="wifi_ap_status_stretch">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="wifi_ap_status_enabled">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="wifi_ap_status_connected_count">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="wifi_ap_status_ip">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="wifi_ap_status_subnet_mask">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="wifi_ap_status_gateway">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="wifi_ap_status_mac_address">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="wifi_ap_status_rx">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="wifi_ap_status_tx">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="wifi_status_vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="wifi_status_button_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wifi_status_button_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
